textproc/chordii/Portfile

```diff
@@ -0,0 +1,52 @@
+PortSystem          1.0
+
+name                chordii
+version             4.5.1
+revision            0
+categories          textproc
+maintainers         breun.nl:nils openmaintainer
+platforms           darwin
+license             GPL-2
+description         Chordii takes input files containing the lyrics and chords of one or more songs and produces a print (PostScript) version of these songs.
+long_description    Chordii, pronounced chord-ee-ee, is a utility that was \
+                    first created by lazy guitarists who got tired of turning \
+                    pages in the middle of the songs they liked. \
+                    \
+                    Chordii takes one or more input files containing the \
+                    lyrics and chords of one or more songs and produces a \
+                    print (PostScript) version of these songs. \
+                    \
+                    The output has the following characteristics: \
+                    * titles and sub-titles have been centered, \
+                    * the lyrics appear in the font and size of your choice, \
+                    * all chords names appear right above the right lyrics, \
+                    * all chords used in a song appear as grids at the bottom of the page. \
+                    \
+                    Optionally, you can also: \
+                    * generate an index of your songs, \
+                    * have the pages numbered, \
+                    * have chords transposed up or down, \
+                    * print in 2-up or 4-up modes (multiple pages per printed sheet) and, \
+                    * insert tablature and comments. \
+                    \
+                    You have a great many options on the final appearance of your songs.
+
+homepage            https://www.vromans.org/johan/projects/Chordii/
+master_sites        https://sourceforge.net/projects/chordii/files/chordii/4.5/
+distname            ${name}-${version}-mac
+worksrcdir          ${workpath}
+
+use_zip             yes
+checksums           rmd160  f2f836a7273d1301a24dc1311002e4752356f14f \
+                    sha256  16e3c63dc5a45112de3fe7d59f0a6a0fc8e66ec99d835e06a214d551f9414965
+
+use_configure       no
+
+build {}
+
+destroot {
+    # Copy over the binaries
+    foreach f { a2crd chordii } {
+        copy ${worksrcpath}/${f} ${destroot}${prefix}/bin
```

Comment:
I didn't check all of those ports, but many of them are related to Java. Java applications are a bit special. Often there are no sources, the binaries are universal in the sense that they run in JVM and thus there's no need to distinguish between OS version and architecture (i386, x86_64, ...). Most of them are problematic in its own way.

But if C sources are available, they should be used.

The other comments are minor and unimportant. This is a way more serious one.